Android/app/google-services.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
just asking, what is the need of this file?

Reply:
Have no idea why and how it got there. Git just included it as i did a ``` git add . ```